lib/node_modules/@stdlib/math/base/special/fresnel/test/test.assign.js

```diff
@@ -0,0 +1,282 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2022 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var isnan = require( '@stdlib/math/base/assert/is-nan' );
+var PINF = require( '@stdlib/constants/float64/pinf' );
+var NINF = require( '@stdlib/constants/float64/ninf' );
+var EPS = require( '@stdlib/constants/float64/eps' );
+var abs = require( '@stdlib/math/base/special/abs' );
+var randu = require( '@stdlib/random/base/randu' );
+var Float64Array = require( '@stdlib/array/float64' );
+var fresnel = require( './../lib/assign.js' );
+
+
+// FIXTURES //
+
+var smallPositive = require( './fixtures/c/cephes/small.json' );
+var mediumPositive = require( './fixtures/c/cephes/medium.json' );
+var largePositive = require( './fixtures/c/cephes/large.json' );
+var hugePositive = require( './fixtures/c/cephes/huge.json' );
+
+
+// TESTS //
+
+tape( 'main export is a function', function test( t ) {
+	t.ok( true, __filename );
+	t.equal( typeof fresnel, 'function', 'main export is a function' );
+	t.end();
+});
+
+tape( 'the function computes in-place the Fresnel integral (small positive values)', function test( t ) {
```

Comment:
```suggestion
tape( 'the function computes Fresnel integrals (small positive values)', function test( t ) {
```

Not computing in-place in these tests.